src/utils/moment.js

```diff
@@ -71,18 +75,26 @@ export default async function loadMomentLocalization() {
  */
 async function getLocaleFor(locale) {
 	try {
-		// default load e.g. fr-fr
+		// default load e.g.en-de
 		await import('moment/locale/' + locale)
 		return locale
 	} catch (error) {
+		const splitLocale = locale.split('-')
 		try {
-			// failure: fallback to fr
-			locale = locale.split('-')[0]
+			// failure: fallback to de
+			locale = splitLocale[1]
```

Comment:
This will probably not go well with special cases of locales such as `ja-Latn-hepburn`.

Reply:
I don´t see how it would, let me illustrate:

First it tries to load 'ja-Latn-hepburn´ and fails.

Then it splits the string into ['ja', 'Latn', 'hepburn']

The second try, it tries to load 'Latn', and fails.

The third try, it tries to load 'ja' and succeeds.

This is why this code should always succeed, as all locales should start with a language tag according to the (the W3 spec)[https://www.w3.org/International/articles/language-tags/#rfc]. The only theoretical situation where it fails I can think of, is a language which has not corresponding Momentjs locale, at which point it should fall back to english. I do now notice that the english fallback does not actually load a moment locale, I will fix that later today